validation/src/main/scala/hmda/validation/dsl/PredicateCommon.scala

```diff
@@ -54,7 +54,8 @@ object PredicateCommon {
   def numeric[T]: Predicate[T] = new Predicate[T] {
     override def validate: (T) => Boolean = _.asInstanceOf[AnyRef] match {
       case n: Number => true
-      case _ => throw new NotImplementedError("'numeric' doesn't handle string (or other) values yet")
+      case s: String => (s forall Character.isDigit) && !s.isEmpty
```

Comment:
True, but I feel like this code could be helpful in the future.  Would it be better to make this change as a separate, one-line pull request (which might satisfy codecov)?